extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts

```diff
@@ -12,6 +12,7 @@ export class RadioGroupLoadingComponentBuilder implements azdata.ComponentBuilde
 	private _optionsLoadingBuilder: azdata.LoadingComponentBuilder;
 	private _onValueChangedEmitter: vscode.EventEmitter<void> = new vscode.EventEmitter();
 	private _currentRadioOption!: azdata.RadioButtonComponent;
+	private _enabled: boolean = false;
```

Comment:
You should set this in the constructor and use the same logic as originally so we use the correct default enabled state for non-dynamic fields 

`this._enabled = instanceOfDynamicEnablementInfo(this._fieldInfo.enabled) ? false : this._fieldInfo.enabled`